homeassistant/components/plugwise/__init__.py

```diff
@@ -110,6 +125,8 @@ async def async_update_data():
     if single_master_thermostat is None:
         platforms = SENSOR_PLATFORMS
 
+    entry.add_update_listener(_update_listener)
```

Comment:
Please store and unsubscribe the listener when the config entry is unloaded to prevent it leaking.  Example in `nut`

Reply:
Adding/testing

Reply:
Hopefully implemented as indicated, passing local functionality and commit-checks.